config-connector/solutions/networking/helm/autoneg/README.md

```diff
@@ -0,0 +1,190 @@
+    ```
+
+1. Switch the context to the cluster that contains the configs for load balancing resources and run verify that multi-cluster ingress is configured
+
+    ```bash
+    # switch the context to the main cluster
+    gcloud container clusters get-credentials [CLUSTER NAME] --zone=[CLUSTER ZONE]
+
+    # if you created the load balancing resources in the namespace, other than default, switch the context to that namespace
+
+    # verify that your backend service has 2 backends attached (select index of "global" if prompted)
+    gcloud compute backend-services describe node-app-backend-service
+    ```
+
+    The backends section of the output should list both backends, for example:
+
+    ```yaml
+    backends:
+    - balancingMode: RATE
+      capacityScaler: 1.0
+      group: https://www.googleapis.com/compute/v1/projects/<project_id>/zones/us-central1-b/networkEndpointGroups/k8s1-37f1db7d-default-node-app-backend-80-486adca6
+      maxRatePerEndpoint: 100.0
+    - balancingMode: RATE
+      capacityScaler: 1.0
+      group: https://www.googleapis.com/compute/v1/projects/<project_id>/zones/europe-west2-a/networkEndpointGroups/k8s1-292a63d7-default-node-app-backend-80-636c84c5
+      maxRatePerEndpoint: 100.0
+      connectionDraining:
+      drainingTimeoutSec: 300
+    ```
+
+    Verify that load balancing resources are forwarding the request to the backend:
+
+    ```bash
+    # curl the external address of the forwarding rule. Note that it might take around 5-10 minutes for load balancing to start working.
+    # You will see the message ("Hello from North America!" or "Hello from Europe!" based on your location).
+    curl $(kubectl get computeforwardingrule -o=jsonpath='{.items[0].spec.ipAddress.addressRef.external}')
+
+1. Clean up the installation:
+
+    ```bash
+    # switch the context to the first cluster. The command uses cluster name and zone based on the values used to create the clusters.
+    gcloud container clusters get-credentials cluster-na --zone=us-central1-b
+
+    # list Helm releases to obtain release name
+    helm list
+
+    # delete release specifying release name from the previous command output. Note that can take a few minutes before all K8s resources are fully deleted.
+    helm delete [release_name]
+    
+     # switch the context to the second cluster. The command uses cluster name and zone based on the values used to create the clusters.
+    gcloud container clusters get-credentials cluster-eu --zone=europe-west2-a
+
+    # list Helm releases to obtain release name
+    helm list
+
+    # delete release specifying release name from the previous command output. Note that can take a few minutes before all K8s resources are fully deleted.
+    helm delete [release_name]
+
+     # switch the context to the cluster that contains the configs for load balancing resources
+    gcloud container clusters get-credentials [CLUSTER NAME] --zone=[CLUSTER ZONE]
+
```

Comment:
Would be great if you also add the step specified in line 128.

Reply:
Added.